spec/parser_spec.lua

```diff
@@ -153,14 +153,22 @@ describe("Pallene parser", function()
     it("can parse programs starting with whitespace or comments", function()
         -- This is easy to get wrong in hand-written LPeg grammars...
         local prog_ast = assert_parses_successfuly("--hello\n--bla\n  ")
-        assert.are.same({}, prog_ast)
+        assert.are.same({}, prog_ast.tls)
     end)
 
     it("can parse toplevel var declarations", function()
-        assert_program_ast([[ local x=17 ]], {
-            { _tag = "ast.Toplevel.Var",
-                decls = { { name = "x", type = false } } }
-        })
+        assert_program_ast([[ local x=17 ]],
```

Comment:
I think it would be more readable if the assert_program_ast continued to receive a list of tls as a parameter. We we could  add the ast.Program.Program tag in the call to `assert_is_subset`.